src/main/scala/scorex/core/network/NodeViewSynchronizer.scala

```diff
@@ -39,22 +40,24 @@ class NodeViewSynchronizer[P <: Proposition, TX <: Transaction[P], SI <: SyncInf
  syncInfoSpec: SIS,
  networkSettings: NetworkSettings) extends Actor with ScorexLogging {
 
-  import History.HistoryComparisonResult._
   import NodeViewSynchronizer._
+  import History.HistoryComparisonResult._
 
   type Timestamp = Long
 
   protected val deliveryTimeout: FiniteDuration = networkSettings.deliveryTimeout
   protected val maxDeliveryChecks: Int = networkSettings.maxDeliveryChecks
   protected val deliveryTracker = new DeliveryTracker(context, deliveryTimeout, maxDeliveryChecks, self)
 
-  /*
+  /**
     We cache peers along with their statuses (whether another peer is ahead or behind of ours,
     or comparison is not possible, or status is not yet known)
-   */
+   **/
   protected val statuses = mutable.Map[ConnectedPeer, HistoryComparisonResult.Value]()
   protected val statusUpdated = mutable.Map[ConnectedPeer, Timestamp]()
-  private var lastSyncInfoSentTime = NetworkTime.time()
+
```

Comment:
Maybe we could encapsulate all this mutable state (e.g. `statuses`, `statusUpdated`) into a class `StatusTracker` (similarly to what we did for `DeliveryTracker`) and then `stableSyncRegime` could be a public `def`, instead of a `var` inside this new class.